tests/nodeLabeller_test.go

```diff
@@ -172,11 +172,11 @@ var _ = Describe("Node Labeller", func() {
 		})
 
 		table.DescribeTable("should restore modified app labels", expectAppLabelsRestoreAfterUpdate,
-			table.Entry("cluster role", &clusterRoleRes),
-			table.Entry("cluster role binding", &clusterRoleBindingRes),
-			table.Entry("security context constraint", &securityContextConstraintRes),
-			table.Entry("Config Map", &configMapRes),
-			table.Entry("daemonSet", &daemonSetRes),
+			table.Entry("[test_id:5818] cluster role", &clusterRoleRes),
+			table.Entry("[test_id:5819] cluster role binding", &clusterRoleBindingRes),
+			table.Entry("[test_id:CNV-5820] security context constraint", &securityContextConstraintRes),
```

Comment:
Looks like you dropped a CNV- in here :-)

Reply:
Thanks @kwiesmueller 
Fix both also in validator_test.go